### DISCUSSION TITLE
How to show PointCloud2 in image panel

### DISCUSSION THREAD

**Discussion by Sirikoon** [foxglove/community#10] [Q&A] [answered]
I have seen in RVIZ that the sensor_msgs/PointCloud2 datatype can be overlay/marker over the camera image.
Is there a way to make PointCloud2 work as a marker in the image panel?

**Comment by jtbandes** [comment, marked as answer]
Not currently, but it is planned. Please follow this ticket for updates: https://github.com/foxglove/studio/issues/2181
